examples/cross-contract-low-level/src/lib.rs

```diff
@@ -1,10 +1,10 @@
 use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
 use near_sdk::json_types::U128;
 use near_sdk::serde_json::{self, json};
-use near_sdk::{env, near_bindgen, AccountId, PromiseResult};
+use near_sdk::{env, Gas, near_bindgen, AccountId, PromiseResult};
 
 // Prepaid gas for making a single simple call.
-const SINGLE_CALL_GAS: u64 = 200000000000000;
+const SINGLE_CALL_GAS: Gas = Gas::new(200000000000000);
```

Comment:
I am undecided between options

* status quo (using `new` everywhere)
* using `Gas(200000000000000)` (without `new`) here and eslwhere in the sdk, to promote this as idiomatic
* not having `Gas::new` at all and requring usage of `Gas(amount)`

Don't think this is an important thing to decide (either option is fine), but I still don't know which one is the best. I have just so slightly bigger pref for the last one, getting rid of `new` entirely.

Reply:
yeah, fair point, probably safer to remove and then can always add back if needed. This was just needed before the inner type was exposed